internal/request/http.go

```diff
@@ -176,3 +204,12 @@ func (h http) isJwtToken() bool {
 
 	return err == nil
 }
+
+func contains(ss []string, s string) bool {
```

Comment:
You can use the `sets.NewString` utility to check if an array contains a value. It's the same logarithmic complexity, it's much more readable.

Reply:
Applied @prometherion, thanks for pointing me out it